README.md

```diff
@@ -23,6 +27,7 @@ You can also download binaries of all versions of `cardano-cli` from [cardano-cl
 Up to date command line help reference is available here:
 * [List of all commands](cardano-cli/test/cardano-cli-golden/files/golden/help.cli)
 * [Description of each command's options](cardano-cli/test/cardano-cli-golden/files/golden/help)
+* [Tutorials](https://developers.cardano.org/docs/get-started/cardano-cli/get-started/)
```

Comment:
That shouln't be a bullet here (it's not a command line reference 😄 ), but it should be at the beginning of the "Documentation" section.